.eslintrc

```diff
@@ -4,7 +4,7 @@
     "@typescript-eslint"
   ],
   "extends": [
-    "wagtail",
+    "@wagtail/eslint-config-wagtail",
```

Comment:
Note: the plugin isn't working as documented: https://www.npmjs.com/package/@wagtail/eslint-config-wagtail